### PR TITLE
Cleanup and documentation

### PR DESCRIPTION
**Pull request by omarhachach**

### CODE REVIEW

File: sso/src/main/java/dk/acto/fafnir/sso/provider/SamlProvider.java [hunk 1]
```diff
@@ -75,6 +78,13 @@ public String getSamlRegistrationIds(String orgId) {
                 .orElseThrow(OrganisationNotUsingSaml::new);
     }
 
+    public List<OrganisationData> getOrgsForProvider() {
```
Comment (FlameDuck) [inline sso/src/main/java/dk/acto/fafnir/sso/provider/SamlProvider.java:81]: Should this be public?

**Review by FlameDuck — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+Ensure that you have JDK 18 and Docker installed. Now, to build you need to run:
+```Bash
+.\gradlew build
+```
+This will build the whole project. Now to run both IAM & SSO, you can run the Compose project:
+```Bash
+docker compose up -d --build
+```
+You should now be able to access the project. By default the Compose project, generates an admin user with the
+credentials: `FAFNIR_ADMIN:pass`. You can just open the [Hazelcast provider](http://localhost:8080/hazelcast/login) and
```
[the remark this answers is not in the document]
Comment (omarhachach) [reply README.md:149]: Maybe I worded it incorrectly, the compose project will by default have the IAM_ADMIN_PASSWORD set to pass and generate the user. The fact that it only generates it is documented above in the Fafnir Configuration section.

[the remark this answers is not in the document]
Comment (omarhachach) [reply README.md:149]: Will fix the default username

File: sso/src/main/java/dk/acto/fafnir/sso/provider/SamlProvider.java (hunk 1 above)
Comment (omarhachach) [reply sso/src/main/java/dk/acto/fafnir/sso/provider/SamlProvider.java:81]: It is used in https://github.com/actoaps/fafnir-sso/pull/86/files/20f1cc65c36f17ac5a37b713470f5b34a30b6a8c#diff-464769c558c7d4ec63d3d261f4bec8d1ccbb535a2924a88b248b76dc2a786fa1R47

**Review by FlameDuck — APPROVED** [review]

**Pull request merged**